metis-enrichment/metis-enrichment-service/src/main/java/eu/europeana/enrichment/service/WikidataAccessService.java

```diff
@@ -190,6 +189,8 @@ public Organization toOrganizationImpl(WikidataOrganization wikidataOrganization
    */
   public void saveXmlToFile(String xml, File contentFile) throws WikidataAccessException {
     try {
+      if(!contentFile.exists())
```

Comment:
Don't need this if statement. File.createNewFile will only create a file if it doesn't already exists.

Reply:
Correct, but the next line of code FileUtils.write() requires that the file exists. I think it is better to create the file in this method, eather than in all other methods that call the serialization to xml file.

Reply:
No, I mean keep the line below (contentFile.createNewFile();) but just remove the line with the if statement. 

contentFile.createNewFile will not create a new file if one already exists. So it does it's own check - no need to add ours.There is no point in checking the disk twice (and generating the extra IO that comes with that).

Reply:
I see ..  I was reading too fast the comment. I remove the line now.